default_api.go

```diff
@@ -665,10 +665,9 @@ func (a *DefaultApiService) CreatePullRequest(projectKey, repositorySlug string,
 Creates a branch using the information provided in the {@link RestCreateBranchRequest request}  &lt;p&gt;  The authenticated user must have &lt;strong&gt;REPO_WRITE&lt;/strong&gt; permission for the context repository to call  this resource.
 
 @return */
-func (a *DefaultApiService) CreateBranch(projectKey, repositorySlug string) (*APIResponse, error) {
+func (a *DefaultApiService) CreateBranch(projectKey, repositorySlug string, localVarPostBody interface{}) (*APIResponse, error) {
```

Comment:
we should remove the `localVar` prefix now that this is an argument

Reply:
Addressed in commit 5eb9c9294cbea3907c5d25aa57adf003ed31bf29